api/container/containerv1/clusters_test.go

```diff
@@ -215,7 +215,18 @@ var _ = Describe("Clusters", func() {
               "ServerURL": "",
               "State": "normal",
               "IsPaid": false,
-              "WorkerCount": 1}`),
+              "WorkerCount": 1,
+              "Vlans": [{
+			  "ID": "177453",
+				"Subnets": [
+				{
+				"Cidr": "159.8.226.208/29",
+				"ID": "1541737",
+				"Ips": ["159.8.226.210"],
+				"IsByOIP": false,
+				"IsPublic": false
+				}]
+				}]}`),
 					),
```

Comment:
How about asserting the values in the test?

Reply:
Did that